packages/css/src/components/progress-list/progress-list.scss

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@use "../../common/hyphenation" as *;
+@use "../../common/text-rendering" as *;
+
+@mixin reset-ol {
+  list-style-type: none;
+  margin-block: 0;
+  padding-inline-start: 0;
+}
+
+.ams-progress-list__connector {
```

Comment:
Please move this selector to between progress (→ indicator) and connector. I think we sort CSS more hierarchically than tokens 😇